### PR TITLE
Optimisation - PART 5

### PR DESCRIPTION
**Pull request by oualib**
- same as previous
 - trying to format correctly the code and erase redundancy.

### CODE REVIEW

File: verticapy/learn/model_selection.py [hunk 1]
```diff
@@ -1773,36 +1776,43 @@ def grid_search_cv(
...
             "final_print" not in kwargs or kwargs["final_print"] != "no_print"
         ):
             print("\033[1mGrid Search Selected Model\033[0m")
             print(
-                f"{str(estimator.__class__).split('.')[-1][:-2]}; Parameters: {result['parameters'][0]}; \033[91mTest_score: {result['avg_score'][0]}\033[0m; \033[92mTrain_score: {result['avg_train_score'][0]}\033[0m; \033[94mTime: {result['avg_time'][0]}\033[0m;"
+                f"{str(estimator.__class__).split('.')[-1][:-2]}; "
+                f"Parameters: {result['parameters'][0]}; \033"
+                "[91mTest_score: {result['avg_score'][0]}\033[0m;"
+                " \033[92mTrain_score: {result['avg_train_score']"
+                "[0]}\033[0m; \033[94mTime: {result['avg_time'][0]}\033[0m;"
```
Comment (mail4umar) [inline verticapy/learn/model_selection.py:1796]: shouldn't there be f in the start of the strings?

File: verticapy/learn/vmodel.py [hunk 2]
```diff
@@ -3896,10 +3857,9 @@ def plot_scree(self, ax=None, **style_kwds):
         ax.set_ylabel('"percentage_explained_variance"')
         ax.set_xlabel('"dimensions"')
         for i in range(n):
+            text_str = f"{round(explained_variance[i], 1)}%"
             ax.text(
-                i + 1.5,
-                explained_variance[i] + 1,
-                "{}%".format(round(explained_variance[i], 1)),
+                i + 1.5, explained_variance[i] + 1, text_str,
```
Comment (mail4umar) [inline verticapy/learn/vmodel.py:3862]: you are removing rounding off

**Review by mail4umar — APPROVED** [review]

File: verticapy/learn/vmodel.py (hunk 2 above)
Comment (oualib) [reply verticapy/learn/vmodel.py:3862]: no, it is in the text_str variable

File: verticapy/learn/model_selection.py (hunk 1 above)
Comment (oualib) [reply verticapy/learn/model_selection.py]: yes, that's a mistake.

**Pull request merged**